test/scenarios/release/release_test.go

```diff
@@ -30,8 +34,19 @@ func TestMain(m *testing.M) {
 }
 
 func TestRelease(t *testing.T) {
+	cmd := exec.Command("git", "describe", "--tags", "--abbrev=0")
+	latestTag, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Fatalf("Failed to run 'git describe': %v", err)
+	}
+
+	usesOldVersion := false
+	usedSemVer, _ := dtversion.ToSemver(string(latestTag))
+	if semver.Compare(thresholdVersion, usedSemVer) >= 1 {
+		usesOldVersion = true
+	}
```

Comment:
will this work in the CI?

if we are still running the things on Concourse in the background then we do not have `git` when we run this.
- Concourse just gets the content of the branch/tag, but doesn't do a full `git clone`, so we can't use git
  - If we could use `git` then we would have done this from the beginning

Reply:
As we run it using the new Github Action Runner maybe you try to read it from envy or so.

@0sewa0 wdyt if we just bump and hardcode version again?

Reply:
as this supposed to be a bugfix, I would actually prefer it, we should revisit the "get the version dynamically" problem in a separate ticket

Reply:
@waodim  ☝️

Reply:
Got it. I will go back to the previous approach.

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/3205/commits/d771f4da34f9527c3661b0cfb31d886a11db4bbf